sncast_std/src/lib.cairo

```diff
@@ -57,9 +58,12 @@ impl DisplayDeclareResult of Display<DeclareResult> {
 }
 
 pub fn declare(
-    contract_name: felt252, max_fee: Option<felt252>, nonce: Option<felt252>
+    contract_name: ByteArray, max_fee: Option<felt252>, nonce: Option<felt252>
 ) -> DeclareResult {
-    let mut inputs = array![contract_name];
+    // it's in fact core::byte_array::BYTE_ARRAY_MAGIC but it can't be imported here
+    let mut inputs = array![0x46a6158a16a947e5916b2a2ca68501a45e93d7110e81aa2d6438b1c57c879a3];
```

Comment:
Why can't it be imported 🤔

Reply:
In cast they use cairo edition `2023_11` which introduces `pub` and `BYTE_ARRAY_MAGIC` is `pub(crate)` in context of `core` so it can't be imported.